brack-plugin/src/plugin.rs

```diff
@@ -104,59 +55,3 @@ pub fn arg_counter(arg_types: &Vec<Type>) -> (usize, usize) {
     }
     (min, max)
 }
-
-#[cfg(test)]
-mod tests {
-    use core::panic;
-
-    use super::*;
-    use brack_parser::ast::new_document;
-    use brack_sdk_rs::{ast::AST, Type, Value};
-    use extism_convert::Json;
-
-    #[test]
-    fn test1() {
-        let mut plugins2 = new_plugins2(vec!["./test.html.wasm"]).unwrap();
-        let (plugin, metadata_map) = plugins2.get_mut("test").unwrap();
-
-        match metadata_map.get(&("@".to_string(), Type::TInline)) {
-            Some(metadata) => {
-                match plugin.call::<Json<Vec<Value>>, String>(
-                    metadata.call_name.clone(),
-                    Json(vec![
-                        Value::Text("https://github.com/momeemt".to_string()),
-                        Value::TextOption(Some("GitHub".to_string())),
-                    ]),
-                ) {
-                    Ok(processed) => assert_eq!(
-                        processed,
-                        "<a href=\"https://github.com/momeemt\">GitHub</a>"
-                    ),
-                    Err(e) => panic!("{:?}", e),
-                }
-            }
-            _ => panic!("No metadata found for @"),
-        }
-    }
-
-    #[test]
-    fn test_macro() {
-        let mut plugins2 = new_plugins2(vec!["./test.html.wasm"]).unwrap();
-        let (plugin, metadata_map) = plugins2.get_mut("test").unwrap();
-
-        let ast = new_document();
-
-        match metadata_map.get(&("^".to_string(), Type::TAST)) {
-            Some(metadata) => {
-                match plugin.call::<Json<(AST, String)>, Json<AST>>(
-                    metadata.call_name.clone(),
-                    Json((ast.clone(), ast.id())),
-                ) {
-                    Ok(Json(ast)) => println!("{:?}", ast),
-                    Err(e) => panic!("{:?}", e),
-                }
-            }
-            _ => panic!("No metadata found for ^"),
-        }
-    }
-}
```

Comment:
I think we need a replacement for the erased test.

Reply:
Your claim is valid, but currently we have to prepare plugin that meets new specifications. It means that we have to manually generate wasm binary every time the specifications change, so I would like to write unit tests after I will make automation testing system for plugins.